app/config/config.yml

```diff
@@ -84,3 +84,15 @@ swiftmailer:
     port:      %sylius.mailer.port%
     username:  %sylius.mailer.user%
     password:  %sylius.mailer.password%
+    spool:     { type: memory }
+
+knp_gaufrette:
+    adapters:
+        exported_files:
+            local:
+                directory:  %sylius.filesystem.export%
+                create:    true
+    filesystems:
```

Comment:
Use should place this config here https://github.com/michalmarcinkowski/Sylius/blob/master/src/Sylius/Bundle/CoreBundle/Resources/config/app/main.yml#L16-16